packages/backend/src/routes/CommentRouter.ts

```diff
@@ -149,6 +148,7 @@ async function createComment(req, res) {
 
     const comment = await req.db.create('Comment', {
         postId,
+        onChainPostId: post.onChainPostId,
```

Comment:
What do you think of removing `onChainPostId` and having `postId` be that value?

It shouldn't be possible to make a comment on a post until the post exists onchain, right?